src/core/server/config/deprecation/core_deprecations.test.ts

```diff
@@ -244,7 +244,49 @@ describe('core deprecations', () => {
       });
       expect(messages).toMatchInlineSnapshot(`
         Array [
-          "\\"logging.events.ops\\" has been deprecated and will be removed in 8.0. To access ops data moving forward, please enable debug logs for the \\"metrics.ops\\" context in your logging configuration.",
+          "\\"logging.events.ops\\" has been deprecated and will be removed in 8.0. To access ops data moving forward, please enable debug logs for the \\"metrics.ops\\" context in your logging configuration. For more details, see https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.md",
+        ]
+      `);
+    });
+
+    it('does not warn when other events are configured', () => {
+      const { messages } = applyCoreDeprecations({
+        logging: { events: { log: '*' } },
+      });
+      expect(messages).toEqual([]);
+    });
+  });
+
+  describe('logging.events.request and logging.events.response', () => {
+    it('warns when request and response events are used', () => {
+      const { messages } = applyCoreDeprecations({
+        logging: { events: { request: '*', response: '*' } },
+      });
+      expect(messages).toMatchInlineSnapshot(`
+        Array [
+          "\\"logging.events.request\\" and \\"logging.events.response\\" have been deprecated and will be removed in 8.0. To access request and/or response data moving forward, please enable debug logs for the \\"http.server.response\\" context in your logging configuration. For more details, see https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.md",
```

Comment:
what if we automatically re-write `logging.events.request: "*"` to 
```yml
  loggers:
     - context: http.server.response
       level: debug
```
and `logging.events.request: "!"` to 
```yml
  loggers:
     - context: http.server.response
       level: off
```
in order to:
- remove all the logic related to request size calculation from `packages/kbn-legacy-logging`
- not to make the user read our documentation

Reply:
I like this idea and how it could simplify things - My only concern would be that it would also break any settings someone had set with the legacy logging filters (`logging.filter.authorization` and `logging.filter.cookie`), since the implementation in this PR simply redacts those headers and isn't configurable in the same way.

Reply:
agree